nimterop/toast.nim

```diff
@@ -9,15 +9,18 @@ proc process(gState: State, path: string, astTable: AstTable) =
 
   var
     parser = tsParserNew()
-    ext = path.splitFile().ext
+    file = path.splitFile()
 
   defer:
     parser.tsParserDelete()
 
   if gState.mode.Bl:
-    if ext in [".h", ".c"]:
+    if '.' in file.name:
+      # triage for autoconf (e.g. '.h.in') until stdlib fix
+      file = file.name.splitFile()
```

Comment:
This is better handled by the wrapper dev - they can use cpFile or mvFile to copy/rename the source file to something else, if they want to use the file raw.

Better yet, they should be using getHeader() to build the library which will generate the final .h files appropriate to the target platform. See nimarchive wrapper for an example of this.